src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -258,17 +288,80 @@ private void EnsureGlobbingUrlBuilder()
             }
         }
 
-        private static void BuildLinkTag(IDictionary<string, string> attributes, TagHelperContent builder)
+        private void BuildLinkTag(IDictionary<string, string> attributes, TagHelperContent builder)
         {
             builder.Append("<link ");
 
             foreach (var attribute in attributes)
             {
-                builder.Append(
-                    string.Format(CultureInfo.InvariantCulture, "{0}=\"{1}\" ", attribute.Key, attribute.Value));
+                if (attribute.Key == "href")
+                {
+                    AppendHref(builder, attribute.Key, attribute.Value);
+                }
+                else
+                {
+                    builder.Append(
+                        string.Format(CultureInfo.InvariantCulture, "{0}=\"{1}\" ", attribute.Key, attribute.Value));
+                }
             }
 
             builder.Append("/>");
         }
+
+        private void AppendHref(TagHelperContent content, string hrefKey, string hrefValue)
+        {
+            string formatter = "{0}=\"{1}";
+            string fileVersion = string.Empty;
+            if (FileVersion == "true")
```

Comment:
It doesn't change the bound properties, it just gives the appearance that things aren't binding because it's not recompiling the pages (they're cached). Going to look into our caching habbits around precompilation today